lib/rules/no-duplicate-imports.js

```diff
@@ -101,6 +101,8 @@ function handleExports(context, importsInFile, exportsInFile) {
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
What happens when there are duplicate imports? If a runtime error would occur, I think this should be a "problem".